aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -561,6 +561,11 @@ private static Boolean isIo1Storage(final ResourceModel model) {
                 (StringUtils.isEmpty(model.getStorageType()) && model.getIops() != null);
     }
 
+    private static Boolean isIo2Storage(final ResourceModel model) {
+        return StorageType.IO2.toString().equalsIgnoreCase(model.getStorageType()) ||
+                (StringUtils.isEmpty(model.getStorageType()) && model.getIops() != null);
```

Comment:
Can you explain what this condition `(StringUtils.isEmpty(model.getStorageType()) && model.getIops() != null)` means? I noticed that it is used in different types like `isGp3Storage`.

Reply:
io1 remnant code. It still needs to be a part of io1 check, but not io2